Tensile/KernelWriterAssembly.py

```diff
@@ -1106,8 +1106,8 @@ def initKernel(self, kernel, tPA, tPB ):
     if kernel["UnrollMajorLDSA"]:
       localReadWidth = (self.lrvwA * tPA["bpe"]) // self.bpr
     # for directToLds x2/x4 support
-    if kernel["DirectToLdsA"]:
-      localReadWidth  = 1    # for fp64 its f32
+    if kernel["DirectToLdsA"] and kernel["GlobalLoadVectorWidthA"] * tPA["bpe"] > 4:
+      localReadWidth  = min(localReadWidth, 1)    # for fp64 its f32
```

Comment:
Do we support LRW less than 1?

Reply:
Yes. 
If UnrollMajorLDSA is false, localReadWidth = tPA["bpe"] / self.bpr.
In f16 case, bpe = 2 and bep / bpr is 0.5.
If we set 1 here, the output code will be inconsistent and it causes a mismatch issue.